test/integration/puppetlabs/general_puppet/general_puppet_int_test.clj

```diff
@@ -259,7 +259,7 @@
 (deftest ^:integration static-file-content-endpoint-test-no-code-content-command
   (logging/with-test-logging
    (testing "the /static_file_content endpoint errors if :code-content-command is not set"
-     (bootstrap/with-puppetserver-running
+     (bootstrap/with-puppetserver-running-with-mock-jrubies
```

Comment:
Sure, if you want to do that, that'd be great.  Or if you want me to do it to justify why I thought it was okay to use mocks in various places, I don't have a problem with that.  Whatever you think is best.